src/js/common/schemaform/helpers.js

```diff
@@ -71,11 +72,12 @@ export function createPageList(formConfig, formPages) {
 export function createRoutes(formConfig) {
   const formPages = createFormPageList(formConfig);
   const pageList = createPageList(formConfig, formPages);
+  const Page = formConfig.disableSave ? FormPage : RoutedSiPPage;
```

Comment:
This is also very minor, but maybe it's safer to actually have `enableSave` in the formConfig vs. defaulting to enabling it by having to specify `disableSave`.

Reply:
We did that because the expectation is that all of the forms will have save in progress, but yeah, maybe the reverse would make sense if non vets teams use this. We can consider changing that later, I think.